### PR TITLE
Electrum 2.0.2

### PR DESCRIPTION
**Pull request by joachifm**
Works for me (UI, wallet creation, and qrcodes tested), whereas the previously packaged version did not.

### CODE REVIEW

**Comment by joachifm** [comment]
Please hold off on merging this until 2.0.3 is released (which should be shortly). I'll update the PR when that happens.

**Comment by joachifm** [comment]
Electrum 2.0.3 has been released.

**Comment by joachifm** [comment]
Looking at the travis log, it seems there's a problem with building pyasn1-modules with pypy.

**Comment by joachifm** [comment]
Disabling the pypy build of pyasn1-modules appears to have fixed the problem. The travis failure is now due to  the log length exceeding the 4mb limit.

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -9943,7 +9943,16 @@ let
 
   ekho = callPackage ../applications/audio/ekho { };
 
-  electrum = callPackage ../applications/misc/electrum { };
+  electrum = callPackage ../applications/misc/electrum {
```
Comment (lethalman) [inline pkgs/top-level/all-packages.nix]: This is quite ugly. What about accepting pythonPackages in the electrum expression.

Comment (joachifm) [inline pkgs/top-level/all-packages.nix]: When I looked at other examples, it seemed most were passing stuff explicitly, but if it is acceptable to pass pythonPackages, then I strongly prefer doing so.

Comment (lethalman) [inline pkgs/top-level/all-packages.nix]: Either put electrum in the python-packages.nix, or pass pythonPackages. Passing explicitly all those packages clutters the definition.

**Comment by joachifm** [comment]
Please let me know if/when you want to merge this, so I can squash it first.

File: pkgs/applications/misc/electrum/default.nix [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
+    protobuf
+    pyasn1
+    pyasn1-modules
+    qrcode
+    requests
+    slowaes
+    tlslite
   ];
 
+  propagatedBuildInputs = with pythonPackages; buildInputs ++ [ pyqt4 ];
```
Comment (lethalman) [inline pkgs/applications/misc/electrum/default.nix]: Why propagated here?

Comment (joachifm) [inline pkgs/applications/misc/electrum/default.nix]: Without it I get "no module named foo" errors at runtime. I initially tried without propagation, but after having had to add 5-6 of the inputs to propagatedInputs, I decided to simply add all of them.
Note that input propagation is present in the old version as well, it's not something I've added, though I tried to get rid off it.

Comment (lethalman) [inline pkgs/applications/misc/electrum/default.nix]: Ok, btw you don't need buildInputs if there's propagatedBuildInputs. A propagated build input is already a build input.

**Comment by joachifm** [comment]
Pushed a squashed version. Thanks for taking the time to look at this @lethalman

**Comment by lethalman** [comment]
Built locally, thanks.

**Pull request merged**